### PR TITLE
Ushcode/repo-moving-tools

### PR DESCRIPTION
**Pull request by Ushcode**
Useful scripts for repository separation

### CODE REVIEW

**Review by Ushcode — COMMENTED** [review]
~This PR also marks as deprecated the old `twa-vis-platform` code and points to its new home~

Above moved to a new PR

**Review by Ushcode — COMMENTED** [review]
Thangks @gpeb2 for the fix on different remote names. Can confirm the script still works as expected on my end.

Also added a prune command to ensure list of branches is synced with the remote as discussed.

Ready to merge once approved

**Review by Ushcode — COMMENTED** [review]
Thangks @gpeb2 for the fix on different remote names. Can confirm the script still works as expected on my end.

Also added a prune command to ensure list of branches is synced with the remote as discussed.

Ready to merge once approved

**Review by gpeb2 — APPROVED** [review]

**Pull request merged**
